voice.go

```diff
@@ -7,22 +7,14 @@ import (
 )
 
 func (c *Client) GetVoiceRegions() ([]*objects.VoiceRegion, error) {
-	res, err := c.request(&request{
-		method:      http.MethodGet,
-		path:        VoiceRegions,
-		contentType: JsonContentType,
-	})
-	if err != nil {
-		return nil, err
-	}
+	regions := []*objects.VoiceRegion{}
+	err := NewRequest().
+		Method(http.MethodGet).
+		Path(VoiceRegions).
+		ContentType(JsonContentType).
+		Expect(http.StatusOK).
+		Bind(regions).
```

Comment:
We need to pass a pointer to the slice here, otherwise it will automatically return an error.